flow-server/src/main/java/com/vaadin/flow/server/VaadinServletService.java

```diff
@@ -240,7 +243,8 @@ public String resolveResource(String url) {
     @Override
     public URL getStaticResource(String path) {
         try {
-            return getServlet().getServletContext().getResource(path);
+            return VaadinServletService
+                    .getStaticResource(getServlet().getServletContext(), path);
```

Comment:
Minor nit: this can just be:
```suggestion
            return getStaticResource(getServlet().getServletContext(), path);
```
As it's a static method in the same class.